src/components/UserList/UserList.js

```diff
@@ -22,8 +23,11 @@ class UserList extends Component {
     this.state = {
 
       // Initialize with mock data for testing listview.
-      dataSource: dataSource.cloneWithRows([1, 2, 3, 4, 5,1, 2, 3, 4, 5,1, 2, 3, 4, 5,1, 2, 3, 4, 5,1, 2, 3, 4, 5,1, 2, 3, 4, 5,1, 2, 3, 4, 5]),
+      dataSource: dataSource.cloneWithRows([1, 2, 3, 4, 5, 1, 2, 3, 4, 5,
```

Comment:
Let's clean this up a little:

      dataSource: dataSource.cloneWithRows([
        1, 2, 3, 4, 5, 1, 2, 3, 4, 5,
        1, 2, 3, 4, 5, 1, 2, 3, 4, 5, 
        1, 2, 3, 4, 5, 1, 2, 3, 4, 5, 
        1, 2, 3, 4, 5,]),